nozzleconfig/nozzle_config.go

```diff
@@ -14,31 +14,32 @@ const (
 )
 
 type NozzleConfig struct {
-	UAAURL                  string
-	Client                  string
-	ClientSecret            string
-	TrafficControllerURL    string
-	FirehoseSubscriptionID  string
-	DataDogURL              string
-	DataDogAPIKey           string
-	HTTPProxyURL            string
-	HTTPSProxyURL           string
-	NoProxy                 []string
-	CloudControllerEndpoint string
-	DataDogTimeoutSeconds   uint32
-	FlushDurationSeconds    uint32
-	FlushMaxBytes           uint32
-	InsecureSSLSkipVerify   bool
-	MetricPrefix            string
-	Deployment              string
-	DeploymentFilter        string
-	DisableAccessControl    bool
-	IdleTimeoutSeconds      uint32
-	AppMetrics              bool
-	NumWorkers              int
-	GrabInterval            int
-	CustomTags              []string
-	DBPath                  string
+	UAAURL                     string
```

Comment:
what happened to this struct?

Reply:
added `DataDogAdditionalEndpoints map[string][]string`